main.cpp

```diff
@@ -4,7 +4,20 @@ using namespace std;
 
 int main()
 {
-  cout << "Hey Mel!";
+  int size;
+  char character;
+  cout << "Enter a size for your triangle";
+  cin  >>  size;
+  cout << "Enter a character: ";
+  cin  >>  character;
+  for(int i = 0; i < size; i++)
+  {
+    for(int j = size; j > i; j--)
+    {
+      cout << character;
```

Comment:
This cout statement should be within an if statement `if ((size - 1) >= j && (size + 1) <= j)`. Otherwise it will just print from the beginning. Also this should have an else component to print out the empty spaces `cout << " ";`.